Core/src/org/sleuthkit/autopsy/guiutils/DataSourceComboBoxModel.java

```diff
@@ -17,7 +17,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.sleuthkit.autopsy.commonfilesearch;
+package org.sleuthkit.autopsy.guiutils;
```

Comment:
Just leaving a note that this move from an non-exported package to an exported package will represent an addition to the public API for this and the other File being moved, it certainly makes more sense with organization though since the new use of this data source list box is not related to common file search.

Reply:
Yes, I did propose this change. But thanks for being alert about public API changes, @wschaeferB!